bftengine/src/bftengine/messages/PrePrepareMsg.hpp

```diff
@@ -68,6 +69,14 @@ class PrePrepareMsg : public MessageBase {
                 const concordUtils::SpanContext& spanContext,
                 size_t size);
 
+  PrePrepareMsg(ReplicaId sender,
```

Comment:
Please don't define additional constructors unless it is absolutely unavoidable. They are hard to maintain. You can give a default value to cid and reuse the existing one.

Reply:
We talked and agreed that the old constructor (without the CID) will call the constructor with the CID (send to_string(seq_num).